CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
Looks like it was unable to load the `VC++` projects. I usually use `2013` as the `VS` version is linked to the `VC++` version (which `VC++ 2013` is the current requirement). Last I checked it was working in `2015`, would have been a few weeks ago the last time I checked.

`CI` server says latest build from `master` was build successfully https://ci.appveyor.com/project/cefsharp/cefsharp/build/55.0.0-CI2059